### ISSUE TITLE
Remove "app/models/spree/order.rb:726 - ensure_customer" warning

### ISSUE THREAD

**Issue by filipefurtad0** [openfoodfoundation/openfoodnetwork#8149]
### What we should change and why (this is tech debt)

A Bugsnag notification is triggered _every_ time the flow to create a backoffice order is started: it is triggered _before_ it is possible to add customer details in the UI.

For this reason, I don't think it is providing any value currently, so I'd propose to remove it - it is causing frequent notifications.

### Context

This was introduced in #7433 to investigate #6038. 

### Impact and timeline

Prevents unnecessary notifications from popping up in Bugsnag / Slack.

**Comment by RachL** [comment]
@filipefurtad0 I could be wrong, but I think you can ask Bugsnag to ignore permanently some errors?

**Comment by filipefurtad0** [comment]
Yes, I think so too @RachL - I'll do that. But I'm wondering if it is a good practice to keep a piece of code which triggers a warning we choose to ignore...
